Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -9994,3 +9994,40 @@ def test_get_engine_base_url(mocker):
     mocker.patch.object(demisto, 'internalHttpRequest', return_value=mock_response)
     res = get_engine_base_url('1111')
     assert res == '11.111.111.33:443'
+
+
+def test_find_and_remove_sensitive_text__found(mocker):
```

Comment:
Lets add some cases, this feels lacking. Lets try with a different regex, when the whole message is sensitive, when regex shows more than once, etc.